doc/user/zebra.rst

```diff
@@ -465,6 +465,76 @@ be updated with the new name. To illustrate, if you want to recompile with
 
 .. _zebra-mpls:
 
+ECMP
```

Comment:
looks like this new section is inserted at the start of the 'mpls' text?

Reply:
yep you are right.  Fixed